NuGet.config

```diff
@@ -9,7 +9,7 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-emsdk -->
-    <add key="darc-pub-dotnet-emsdk-51bf18a" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-51bf18a2/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-2406616" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-2406616d/nuget/v3/index.json" />
```

Comment:
do we need this??? (And the version change bellow)

Reply:
probably mistakenly added more commits because I backported to release/8.0 instead of releaes/8.0-staging. should be fixed now.